### DISCUSSION TITLE
Demo Page Not Loading

### DISCUSSION THREAD

**Discussion by CherryCreekTech** [GrapesJS/grapesjs#4923] [Q&A]
I installed this from Git and the demo page isn't loading.

Do I need to edit code?
Where is the demo.htm and demo_002.htm trying to load in the code?
Can't see any errors, just getting strange behavior.
Canvas looks blank, but there are things loaded for the demo so it's not actually blank.
What happened to the delete all button? That would solve this issue, but you removed it.

TIA for any help.

**Comment by artf** [comment]
Hey @BristleconeStudio if you're looking for the code of the demos you can find them in this branch https://github.com/GrapesJS/grapesjs/tree/gh-pages

**Comment by CherryCreekTech** [comment]
Thank you. I'll use that for testing.
